source/game.database.bmx

```diff
@@ -2119,6 +2124,13 @@ Type TDatabaseLoader
 			Local jobCountry:String = xml.FindValueLC(nodeJob, "country", "")
 			'for actor jobs this defines if a specific role is defined
 			Local jobRoleGUID:String = xml.FindValueLC(nodeJob, "role_guid", "")
+			Local jobRandomRole:Int = 0
+			If jobRoleGUID = "randomRole"
```

Comment:
`jobRoleGUID.ToLower() = "randomrole"`

Reply:
War nur Proof of concept - neues Attribut eingeführt.